### PR TITLE
Add windows download panel

### PR DESCRIPTION
**Pull request by sandersonet**
@gib - I had to hand-craft the artisanal 5 column layout since I don't think neat supports one out of the box.  Let me know if I'm mistaken or if there is a better way. 

screenshot 2017-01-06 09 35 11

cc @krallin

### CODE REVIEW

**Comment by krallin** [comment]
Thanks @sandersonet! 

One suggestion: do you think we should reference "Windows 8.1 or greater" and "Windows Server 2012r2 or greater"? The "and up" wording feels a little off in the context of "requirements".

**Comment by sandersonet** [comment]
Updated to 

> Requires 64 bit Windows 8.1 or greater or 64 bit Windows Server 2012r2 or greater

**Comment by krallin** [comment]
Thank you! ❤️

File: source/stylesheets/_grid.scss [hunk 1]
```diff
@@ -199,6 +199,28 @@ body {
...
+
+    @media screen and (max-width: $tablet-landscape) {
+      @include span-columns(6);
+      @include omega(2n);
+    }
+    @media screen and (max-width: $tablet-portrait) {
+      @include span-columns(12);
+    }
+  }
+}
```
Comment (gib) [inline source/stylesheets/_grid.scss]: 👌 Looks great -- Thanks for sticking with these grid patterns @sandersonet.

**Pull request merged**